internal/tracing/manifests/tracing.go

```diff
@@ -26,23 +23,5 @@ func buildSecrets(resources Options) ([]SecretValue, error) {
 }
 
 func buildOtelColSpec(resources Options) (*otelv1beta1.OpenTelemetryCollectorSpec, error) {
```

Comment:
You can just remove this method and set the value of `&resources.OpenTelemetryCollector.Spec` to the variable.